client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/InteractionDialog/InteractionEventForm/InteractionDetailsForm.tsx

```diff
@@ -53,16 +58,35 @@ const InteractionDetailsForm = (props: Props) => {
         }
     };
 
+    const addFamilyMemberContacts = (contacts: Contact[]) => {
+        familyMembers?.map((familyMember: InvolvedContact) => {
```

Comment:
I dont think map is the right choice here, maybe use reduce or for each (reduce is better)